### PR TITLE
ELEMENTS-1554: improve contrast for select all action

### PR DESCRIPTION
**Pull request by poonamyadav252**

### CODE REVIEW

**Review by swarnadipa-dev — APPROVED** [review]
Looks good

File: ui/widgets/nuxeo-checkmark.js [hunk 1]
```diff
@@ -54,11 +54,12 @@ import '../nuxeo-icons.js';
             height: var(--nuxeo-checkmark-height, 18px);
             cursor: pointer;
             border-radius: 50%;
-            border: 2px solid var(--nuxeo-checkmark-border-color, var(--nuxeo-border, gray));
+            border: 2px solid var(--nuxeo-checkmark-border-color, var(--nuxeo-text-default, gray));
             background-color: var(--nuxeo-checkmark-background-color, transparent);
             color: var(--nuxeo-icon-color, transparent);
             padding: 0;
             margin: 0;
+            opacity: 0.7;
```
Comment (GuruprasadAnk1996) [inline ui/widgets/nuxeo-checkmark.js:62]: Looks too dark in the UI, 3:1 ratio would be enough as mentioned in https://jira.nuxeo.com/browse/ELEMENTS-1554

**Review by GuruprasadAnk1996 — COMMENTED** [review]
Kindly check with opacity rate

File: ui/widgets/nuxeo-checkmark.js (hunk 1 above)
Comment (poonamyadav252) [reply ui/widgets/nuxeo-checkmark.js:62]: opacity less than 7 doesn't work here.
image

**Review by GuruprasadAnk1996 — APPROVED** [review]

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by rajatgoel-gl — APPROVED** [review]

**Pull request merged**